frontend/src/components/VSourcesTable.vue

```diff
@@ -167,37 +191,79 @@ export default defineComponent({
 @tailwind utilities;
 
 @layer components {
-  .table {
-    @apply rounded-sm border-0 border-dark-charcoal-20;
+  .source-table {
+    @apply hidden rounded-sm border-0 border-dark-charcoal-20 md:table;
   }
-  .table th,
-  .table td {
+
+  .source-table th,
+  .source-table td {
     @apply border-dark-charcoal-20;
   }
-  .table a {
+
+  .source-table a {
     @apply text-pink hover:underline;
   }
-  .table th {
+
+  .source-table th {
     @apply cursor-pointer border-t bg-dark-charcoal-10;
   }
-  .table th,
-  .table td {
+
+  .source-table th,
+  .source-table td {
     @apply border-r p-4 first:border-l;
   }
-  .table td {
+
+  .source-table td {
     @apply break-normal border-y-0;
   }
 
-  .table tr {
+  .source-table tr {
     @apply even:bg-dark-charcoal-06;
   }
 
-  .table th {
+  .source-table th {
     @apply first:rounded-ss-sm last:rounded-se-sm;
   }
 
-  .table tr:last-child td {
+  .source-table tr:last-child td {
     @apply border-b first:rounded-es-sm last:rounded-ee-sm;
   }
+
+  .mobile-source-table {
+    @apply md:hidden;
+  }
```

Comment:
Just a nit-pick, but it might be easier to understand the template if these styles were on the element's `class` attribute directly, rather than displaced into the styles.

I think the reason some of these are displaced is to avoid repeating the styles for elements that appear multiple times as children of the root element of the table.

For styles that aren't targeting a child-element in that way, it's probably more consistent and easier to understand the template if those styles are directly on the elements (especially true for visibility/display directives).